types/sumoselect/index.d.ts

```diff
@@ -0,0 +1,56 @@
+// Type definitions for sumoselect 3.0
+// Project: https://github.com/HemantNegi/jquery.sumoselect#readme
+// Definitions by: nagamejun <https://github.com/nagamejun>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="jquery" />
+
+type SumoSelectEvents = 'sumo:opening' | 'sumo:opened' | 'sumo:closing' | 'sumo:closed';
+
+type Option = Partial<{
+    placeholder: string;
+    csvDispCount: number;
+    captionFormat: string;
+    captionFormatAllSelected: string;
+    floatWidth: number;
+    forceCustomRendering: boolean;
+    nativeOnDevice: string[];
+    outputAsCSV: boolean;
+    csvSepChar: string;
+    okCancelInMulti: boolean;
+    isClickAwayOk: boolean;
+    triggerChangeCombined: boolean;
+    selectAll: boolean;
+    search: boolean;
+    searchText: string;
+    noMatch: string;
+    prefix: string;
+    locale: any[];
+    up: string;
```

Comment:
this should be boolean

https://github.com/HemantNegi/jquery.sumoselect/blob/master/jquery.sumoselect.js#L50

```suggestion
    up: boolean;
```